.vsts-ci.yml

```diff
@@ -182,47 +143,3 @@ stages:
         -TsaRepositoryName "dotnet-sdk"
         -TsaCodebaseName "dotnet-sdk"
         -TsaPublish $True'
-
-- stage: helix_tests
```

Comment:
@wli3 do we want the helix tests removed from this branch?

Reply:
these are out dated helix **perf** tests. So this is fine